app/graphql/resolvers/task.rb

```diff
@@ -0,0 +1,12 @@
+module Resolvers
+  class Tasks < Resolvers::Base
+    argument :id, ID, require: true
+
+    type Types::TaskType, null: true
+
+
+    def resolve(**_options)
```

Comment:
We use an underscore at the beginning of the argument name when it is not used in the method